spec/miscellaneous/helpers/mailer_helper_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe MailerHelper do
+  describe "style_creation_link" do
+    it "nests red link inside bold inside italics" do
+      work = create(:posted_work)
```

Comment:
If we can get away with not saving the model (`build(:posted_work)`), we should do that instead.

Reply:
We need it saved for the route, unfortunately, so it fails like this if I use `build`:

```
  1) MailerHelper style_creation_link nests red link inside bold inside italics
     Failure/Error: expect(style_creation_link(work.title, work_url(work))).to eq("<i><b><a style=\"color:#990000\" href=\"#{work_url(work)}\">#{work.title}</a></b></i>")
     
     ActionController::UrlGenerationError:
       No route matches {:action=>"show", :controller=>"works", :id=>nil}, missing required keys: [:id]
     # ./spec/miscellaneous/helpers/mailer_helper_spec.rb:7:in `block (3 levels) in <top (required)>'

Finished in 1.47 seconds (files took 6.14 seconds to load)
1 example, 1 failure
```

Reply:
I forgot it needs the ID. This is good as is.